packages/tokens/README.md

```diff
@@ -7,9 +7,9 @@ Direct consumption of the tokens package is optional, but recommended. The token
 
 To use the tokens, simply:
```

Comment:
Should we also add a section for using themed tokens? 

At face value, seems preferable for apps to also use the themed versions assuming all apps should want/have a light/dark mode. 

Also, bigger picture, wonder if we should make a ticket (epic?) about providing a Theme wrapper. Not a big deal for flagship as they already have it, but for any new apps a plug and play theming system would be a good shortcut to getting up and running quickly.

Reply:
Added section to README for tokens.

Originally planned to do a wrapper but found it was way simpler to create a hook that was based on `useColorScheme()`. Exporting `useTheme()` wasn't part of the scope of this ticket but I tried a quick test of exporting it and importing it in the app but ran into some errors. I'm not sure that the app team would have any desire to switch over to using our themes any time soon given how widely used their existing themes currently are, but I've created a separate ticket to address exporting the hook: #377